Custom Url Schemes/README.md

```diff
@@ -103,9 +103,9 @@ At the end of the file, before the closing tags `</dict></plist>`, insert the fo
 </array>
 ```
 
-**Notice** the string `lumapps-vscode`, it's your custom scheme.
+**Notice** the string `lumapps-vscode`, this is your custom scheme.
```

Comment:
Can you please apply this change to the two other Operating Systems?

Reply:
done